gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/FhirIDGeneratorService.java

```diff
@@ -0,0 +1,12 @@
+package uk.nhs.adaptors.pss.translator.service;
+
+import java.util.UUID;
+
+public class FhirIDGeneratorService {
```

Comment:
You could annotate this class with `@Service` spring annotation, and then you'll be able to use it inside BundleGenerator, just by adding a field `private final FhirIDGeneratorService fhirIdGeneratorService`.